Adjust the CFLAGS in the Solaris Studio Makefile to not include build
options like '-fast' and '-lpthread', use -D_REENTRANT instead of -mt
and to not set a run path.

--- pbzip2-1.1.6/Makefile.solaris.sunstudio.orig	2013-03-18 08:53:39.894869460 -0700
+++ pbzip2-1.1.6/Makefile.solaris.sunstudio	2013-03-18 10:37:02.531235184 -0700
@@ -4,11 +4,12 @@
 # Compiler to use
 CC=CC
 CFLAGS = -m64
-CFLAGS += -fast -xO5
+CFLAGS += -xO5
 
 #CFLAGS += -Wall
 #CFLAGS += -g
-CFLAGS += -mt -lpthread
+CFLAGS += -D_REENTRANT
+CFLAGS += -norunpath
 
 # Comment out CFLAGS line below for compatability mode for 32bit file sizes
 # (less than 2GB) and systems that have compilers that treat int as 64bit
@@ -30,7 +31,6 @@
 
 # Libraries
 LDFLAGS = -lbz2
-LDFLAGS += -lpthread
 
 # Where you want pbzip2 installed when you do 'make install'
 PREFIX = /usr
